tests-integration/tests/main.rs

```diff
@@ -17,5 +17,9 @@ mod grpc;
 #[macro_use]
 mod http;
 
-grpc_tests!(File, S3, S3WithCache, Oss, Azblob);
-http_tests!(File, S3, S3WithCache, Oss, Azblob);
```

Comment:
@WenyXu 

Why did you remove `Azblob`?

I'll add tests for #1611, but `Gcs` is also unnecessary?

Reply:
Good catch! I think that was because I didn't rebase the PR to the latest develop branch. I'm going fix it 👀 Thks @NiwakaDev